### ISSUE TITLE
incrond should accept only certain incrontab filenames (e.g. SOME.conf)

### ISSUE THREAD

**Issue by fhanzlik** [ar-/incron#27]
IMO it would be very helpful if incron accept only certain file names - perhaps names with '.conf' suffix, as is usual with files in X11/xorg.conf.d, dovecot/conf.d, exports.d, modprobe.d, php.d, httpd/conf.d, rsyslog.d, sane.d, sysctl.d, tmpfiles.d, udev/rules.d and others.

**Comment by djstauffer** [comment]
Yes.  This needs to be changed, and here's why:  If anyone uses vi ( or vim with backup files turned on ) to edit the file, the backup file which is left behind {filename}~ will *also* be processed as an incrontab file.  This has bitten me and others at work.
